src/frontends/common/src/manager.cpp

```diff
@@ -106,6 +106,8 @@ class FrontEndManager::Impl {
     }
 
     void register_front_end(const std::string& name, const std::string& library_path) {
+        OPENVINO_ASSERT(!ov::util::is_symlink_in_different_path(library_path),
+                        "Cannot registe plugin with symlink path:\"" + library_path + "\".");
```

Comment:
```suggestion
                        "Cannot register plugin with symlink path: \"" + library_path + "\".");
```
and the same in other places